<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LAB_12_14Class</class>
 <widget class="QMainWindow" name="LAB_12_14Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LAB_12_14</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="myTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>611</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>What are you?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="usergo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>ADMIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="admingo">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>USER</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="goback">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Goback</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listaadmin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>341</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteDog">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete a dog</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adaugafra">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add dog</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateDog">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Dog</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Name">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Breed">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Link">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="Age">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="Weight">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Breed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>link</string>
      </property>
     </widget>
     <widget class="QLabel" name="Error">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>481</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Detailed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Short</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QListWidget" name="listaadopt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="goback_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Goback</string>
      </property>
     </widget>
     <widget class="QListWidget" name="Adopt">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="get">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Adopt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="minw">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="Ag">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="filter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gow">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Return to adoption</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HTML">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>151</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>HTML</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CSV">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>CSV</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LAB_12_14.qrc"/>
 </resources>
 <connections/>
</ui>
